shinhye/services/postService.js

```diff
@@ -0,0 +1,130 @@
+require("dotenv").config();
+const jwt = require("jsonwebtoken");
+const postDao = require("../models/postDao");
+
+const secretKey = process.env.SECRET_KEY;
+
+const createPost = async (title, content, token) => {
+  try {
+    const header = jwt.decode(token);
+    // if (!header.email) return
+    const headerEmail = header.email;
+    const verifyToken = jwt.verify(
```

Comment:
verifyToken 보다는 토큰을 검증하였을 때의 결과값은 토큰을 `복호화` 했을 때의 payload 정보이기 떄문에 `decoded` 혹은 payload에 포함되어 있는 user 정보를 가리키는 `user`라는 변수명을 사용하는 것이 더 좋아보입니다.

해당 코드도 앞서 피드백 드린 사항에 포함하여 모듈로 분리해주면 좋을 것 같네요!

 ```suggestion
    const decoded = jwt.verify(
```